pom.xml

```diff
@@ -532,6 +532,11 @@
                     <artifactId>build-helper-maven-plugin</artifactId>
                     <version>3.0.0</version>
                 </plugin>
+                <plugin>
+                    <groupId>net.wasdev.wlp.maven.plugins</groupId>
+                    <artifactId>liberty-maven-plugin</artifactId>
+                    <version>2.2</version>
```

Comment:
Shall we define a property for this one also?

Reply:
fixed with 7340ce6